### PR TITLE
Add some links around the bazel toolchain topic

### PR DESCRIPTION
**Pull request by Wetrain**
Hi,

I've been looking at bazel toolchain stuff and thought i would add some links here that i found

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -718,6 +728,9 @@ Tools for generating WORKSPACE and BUILD files from source code.
 - [Android](https://docs.bazel.build/versions/master/tutorial/android-app.html)
 - [iOS](https://docs.bazel.build/versions/master/tutorial/ios-app.html)
 
+#### Toolchains
+- [c++ tolchains](https://docs.bazel.build/versions/master/tutorial/cc-toolchain-config.html)
```
Comment (jin) [inline README.md]:
```suggestion
- [C++ toolchains](https://docs.bazel.build/versions/master/tutorial/cc-toolchain-config.html)
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
 
 - [Rules](#rules)
 - [Tooling](#tooling)
+  - [Toolchains](#toolcahins)
```
Comment (jin) [inline README.md]:
```suggestion
  - [Toolchains](#toolchains)
```

File: README.md [hunk 3]
```diff
@@ -634,6 +635,15 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
 - [philwo/bazelisk](https://github.com/philwo/bazelisk) - A user-friendly launcher for Bazel
 - [bzl-io/bzl](https://github.com/bzl-io/bzl) - a wrapper around the bazel build tool to extend its functionality
 
+### Toolchains
+
+Toolchains -  a way for rule authors to decouple their rule logic from platform-based selection of tools.
```
Comment (jin) [inline README.md]:
```suggestion
Toolchains enable rule authors to decouple their rule logic from platform-based selection of tools.
```

File: README.md [hunk 4]
```diff
@@ -634,6 +635,15 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
...
+
+- [bazelbuilds/bazel-toolchains](https://github.com/bazelbuild/bazel-toolchains) - is a repository where Google hosts Bazel toolchain configs to enable Remote Execution via Docker containers
```
Comment (jin) [inline README.md]:
```suggestion
- [bazelbuilds/bazel-toolchains](https://github.com/bazelbuild/bazel-toolchains) - Bazel toolchain configurations to enable Google Cloud Remote Build Execution via Docker containers
```

**Comment by Wetrain** [comment]
@jin Thanks for the quick review! :) I think all of those should be fixed up now

**Comment by jin** [comment]
thanks!

**Pull request merged**